media-sound/freac/additional-files/freac.rdef.in

```diff
@@ -0,0 +1,69 @@
+resource app_signature "application/x-vnd.freac-freac";
```

Comment:
When I start the app from the command line I get this warning:
```
Signature in rsrc doesn't match constructor arg. (application/x-vnd.smooth-haiku, application/x-vnd.freac-freac)
```

Reply:
This is because there is no way yet for the app to tell its signature to the smooth framework. I still need to find a good solution for this and hope I can implement it in the next release.

I hope this is not a blocker for this version. It seems to have no consequences aside from the message when starting from the command line.